test/e2e/parallel/spacebindingrequest_test.go

```diff
@@ -0,0 +1,288 @@
+			// wait for the namespace to be provisioned since we will be creating the SpaceBindingRequest into it.
+			space, err := hostAwait.WaitForSpace(t, space.Name, UntilSpaceHasAnyProvisionedNamespaces())
+			require.NoError(t, err)
+			// let's create a new MUR that will have access to the space
+			username := uuid.Must(uuid.NewV4()).String()
+			_, mur := NewSignupRequest(awaitilities).
+				Username(username).
+				Email(username + "@acme.com").
+				ManuallyApprove().
+				TargetCluster(memberAwait).
+				RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
+				NoSpace().
+				WaitForMUR().Execute(t).Resources()
+			// create the spacebinding request
+			spaceBindingRequest := CreateSpaceBindingRequest(t, awaitilities, memberAwait.ClusterName,
+				WithSpecSpaceRole("invalid"), // set invalid spacerole
+				WithSpecMasterUserRecord(mur.GetName()),
+				WithNamespace(GetDefaultNamespace(space.Status.ProvisionedNamespaces)),
+			)
+
+			// then
+			require.NoError(t, err)
+			// wait for spacebinding request status to be set
+			_, err = memberAwait.WaitForSpaceBindingRequest(t, types.NamespacedName{Namespace: spaceBindingRequest.GetNamespace(), Name: spaceBindingRequest.GetName()},
+				UntilSpaceBindingRequestHasConditions(spacebindingrequesttestcommon.UnableToCreateSpaceBinding(fmt.Sprintf("invalid role 'invalid' for space '%s'", space.Name))),
+			)
+			require.NoError(t, err)
+			bindings, err := hostAwait.ListSpaceBindings(space.Name)
+			require.NoError(t, err)
+			assert.Len(t, bindings, 1)
+		})
+
+		t.Run("unable create space binding request with invalid MasterUserRecord", func(t *testing.T) {
+			space, _, _ := CreateSpace(t, awaitilities, testspace.WithTierName("appstudio"), testspace.WithSpecTargetCluster(memberAwait.ClusterName))
+			// wait for the namespace to be provisioned since we will be creating the SpaceBindingRequest into it.
+			space, err := hostAwait.WaitForSpace(t, space.Name, UntilSpaceHasAnyProvisionedNamespaces())
+			require.NoError(t, err)
+			// create the spacebinding request
+			spaceBindingRequest := CreateSpaceBindingRequest(t, awaitilities, memberAwait.ClusterName,
+				WithSpecSpaceRole("admin"),
+				WithSpecMasterUserRecord("invalidMUR"), // we set an invalid MUR
+				WithNamespace(GetDefaultNamespace(space.Status.ProvisionedNamespaces)),
+			)
+
+			// then
+			require.NoError(t, err)
+			// wait for spacebinding request status to be set
+			_, err = memberAwait.WaitForSpaceBindingRequest(t, types.NamespacedName{Namespace: spaceBindingRequest.GetNamespace(), Name: spaceBindingRequest.GetName()},
+				UntilSpaceBindingRequestHasConditions(spacebindingrequesttestcommon.UnableToCreateSpaceBinding("unable to get MUR: MasterUserRecord.toolchain.dev.openshift.com \"invalidMUR\" not found")),
+			)
+			require.NoError(t, err)
+			bindings, err := hostAwait.ListSpaceBindings(space.Name)
```

Comment:
I believe it would make more sense to check the number of space bindings first, but it's just a minor suggestion

Reply:
sorry, you mean checking the number of spacebindings before waiting for the `SpaceBindingRequest` to have the expected conditions ? Not sure I understood your suggestion ..

Reply:
ah never mind, I mixed up space bindings and space requests 🤦‍♂️

Reply:
Ok, no worries, just wanted to be sure!